### PR TITLE
Creating AWS API Gateway entry for a lambda in F#

### PR DESCRIPTION
**Pull request by marklam**
Noob example of creating an API gateway from Fsharp so that a Giraffe web server can be hosted in an AWS lambda. Based on the aws-cs-lambda example and the serverless.Giraffe Aws template

### CODE REVIEW

File: aws-fs-lambda-webserver/LambdaWebServer/appsettings.Development.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  }
+}
```
Comment (mikhailshilkov) [inline aws-fs-lambda-webserver/LambdaWebServer/appsettings.Development.json]: Can we live without this file?

File: aws-fs-lambda-webserver/LambdaWebServer/aws-lambda-tools-defaults.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  ],
+  "profile"     : "",
+  "region"      : "",
+  "configuration" : "Release",
+  "framework"     : "netcoreapp2.1",
+  "s3-prefix"     : "LambdaWebServer/",
+  "template"      : "serverless.template",
+  "template-parameters" : "",
+  "s3-bucket"           : ""
+}
```
Comment (mikhailshilkov) [inline aws-fs-lambda-webserver/LambdaWebServer/aws-lambda-tools-defaults.json]: Do we need this file?

File: aws-fs-lambda-webserver/LambdaWebServer/serverless.template [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		}
+	},
+
+	"Outputs" : {
+		"ApiURL" : {
+			"Description" : "API endpoint URL for Prod environment",
+			"Value" : { "Fn::Sub" : "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/" }
+		}
+	}
+}
```
Comment (mikhailshilkov) [inline aws-fs-lambda-webserver/LambdaWebServer/serverless.template]: Pretty sure we don't need this file

File: aws-fs-lambda-webserver/LambdaWebServer/appsettings.Development.json (hunk 1 above)
Comment (marklam) [reply aws-fs-lambda-webserver/LambdaWebServer/appsettings.Development.json]: They were created by the AWS template I guess. I've removed them

**Comment by marklam** [comment]
There's a readme in there now too.

**Pull request merged**